llm/memory.go

```diff
@@ -149,7 +149,12 @@ func EstimateGPULayers(gpus []discover.GpuInfo, f *ggml.GGML, projectors []strin
 	}
 
 	if graphPartialOffload == 0 {
-		graphPartialOffload = f.KV().GQA() * kvTotal / 6
+		headsKV := f.KV().HeadCountKVMin()
+		if headsKV == 0 {
+			headsKV = 1
+		}
```

Comment:
`HeadCountKVMin` sets a default of 1

```suggestion
		headsKV := f.KV().HeadCountKVMin()
```

Reply:
the actual minimum value can be 0 though, so I think we still need to guard it here (and if we do, is setting it to 1 the right thing to do here as well?) For example https://huggingface.co/bartowski/nvidia_Llama-3_3-Nemotron-Super-49B-v1-GGUF has

```
heads="[64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 64 0 64 64 0 64 0 64 64 0 64 64 0 0 64 0 0 64 64 64 64 64 64 64 64 64 64 64 64 64 64]" headsKV="[8 4 8 8 8 2 2 1 1 2 2 0 1 2 2 0 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 8 0 8 8 0 0 0 0 0 0 0 8 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 8 8 8 8 8 8 8 8 8 8]"
```